runtime/compiler/env/VMJ9.cpp

```diff
@@ -5979,7 +5980,7 @@ TR_J9VMBase::getStringClassEnableCompressionFieldAddr(TR::Compilation *comp, boo
          TR_PersistentClassInfo * classInfo = (comp->getPersistentInfo()->getPersistentCHTable() == NULL) ?
             NULL :
             comp->getPersistentInfo()->getPersistentCHTable()->findClassInfoAfterLocking(stringClass, comp, isVettedForAOT);
-         if (classInfo && classInfo->isInitialized())
+         if (classInfo && classInfo->isInitialized(false))
```

Comment:
Suggestion: instead of hard-coding 'false' here, we could say:
`isInitialized(comp->compileRelocatableCode() && !comp->reloRuntime()->isLoading())`
with a comment say that validation only needs to happen during an AOT compilation, but not during the relocation stage.

Reply:
I could do that, but I'm not sure if it makes sense to do so. The method is set up to cache its result, so only one invocation of this method is ever going to reach this point and create a relo record. Maybe if I were to make that change it would be better not to use the cached result if we are performing an AOT compilation, which would allow for the appropriate relo records to be created, if that makes sense.

Reply:
I am fine with leaving `false` for the parameter. My intent was to actually explain the conditions under which validation in `isInitialized()` is required/not-required (i.e.  only perform validation during an AOT compilation, but not during the relocation phase)